ui/app/AppLayouts/Chat/panels/UserListPanel.qml

```diff
@@ -75,66 +76,127 @@ Item {
                     }
-                    name: "preferredDisplayName"
-                    expectedRoles: ["localNickname", "ensName", "displayName", "alias"]
-                    expression: displayNameProxy(model.localNickname, model.ensName, model.displayName, model.alias)
                 }
 
-                sorters: [
-                    RoleSorter {
-                        roleName: "onlineStatus"
-                        sortOrder: Qt.DescendingOrder
-                    },
-                    StringSorter {
-                        roleName: "preferredDisplayName"
-                        caseSensitivity: Qt.CaseInsensitive
+                Timer {
+                    id: userListDelay
+                    repeat: false
+                    running: false
+                    interval: 500
+                    onTriggered: {
+                        listLoader.active = true
+                        loadingUsersView.active = false
                     }
-                ]
-            }
-            section.property: "onlineStatus"
-            section.delegate: (root.width > 58) ? sectionDelegateComponent : null
-            delegate: StatusMemberListItem {
-                width: ListView.view.width
-                nickName: model.localNickname
-                userName: ProfileUtils.displayName("", model.ensName, model.displayName, model.alias)
-                pubKey: model.isEnsVerified ? "" : Utils.getCompressedPk(model.pubKey)
-                isContact: model.isContact
-                isVerified: model.isVerified
-                isUntrustworthy: model.isUntrustworthy
-                isAdmin: model.memberRole === Constants.memberRole.owner
-                asset.name: model.icon
-                asset.isImage: (asset.name !== "")
-                asset.isLetterIdenticon: (asset.name === "")
-                asset.color: Utils.colorForColorId(model.colorId)
-                status: model.onlineStatus
-                ringSettings.ringSpecModel: model.colorHash
-                onClicked: {
-                    if (mouse.button === Qt.RightButton) {
-                        Global.openMenu(profileContextMenuComponent, this, {
-                                            myPublicKey: userProfile.pubKey,
-                                            selectedUserPublicKey: model.pubKey,
-                                            selectedUserDisplayName: nickName || userName,
-                                            selectedUserIcon: model.icon,
-                                        })
-                    } else if (mouse.button === Qt.LeftButton) {
-                        Global.openProfilePopup(model.pubKey);
+                }
+
+                sourceComponent: StatusListView {
+                    id: userListView
+                    objectName: "userListPanel"
+
+                    clip: false
+
+                    visible: listLoader.chatId === root.activeChatId
```

Comment:
Loading something only to hide it later? Sounds fishy :)

Reply:
Similar explanation as above. We keep 5 loaded and hide the ones that are not active.

Reply:
Yeah I see, thanks :)